src/ConnectedMode/Persistence/BindingDto.cs

```diff
@@ -0,0 +1,37 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Newtonsoft.Json;
+using SonarLint.VisualStudio.Core;
+using SonarLint.VisualStudio.Core.Binding;
+using SonarQube.Client.Models;
+
+namespace SonarLint.VisualStudio.ConnectedMode.Persistence;
+
+[JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+internal class BindingDto
```

Comment:
I would rather call it "BindingJsonModel" or "BindingJsonDto" to differentiate it between other dtos (used, for example, to communicate with SlCore) and one used for json

Reply:
Yeah I wanted to do that, but then thought maybe json is too specific. I can rename it, sure